torch/nn/parallel/distributed.py

```diff
@@ -1385,9 +1433,22 @@ def _lazy_init(self):
         self._setup_in_backward_optimizers()
         self._lazy_init_ran = True
 
+    def _should_disable_reducer(self) -> bool:
+        return torch._utils.is_compiling() or self._force_to_disable_reducer
```

Comment:
add a comment here? - why the 'or' (which conditions are each half of the or true, and why isn't one condition sufficient)

Reply:
oh.. i think this is just for testing?  maybe name it `_disable_reducer_for_testing`?

Reply:
This is not for testing. We have to disable the cpp reducer if we enable Python reducer. Change the name to `_should_disable_cpp_reducer`.